### PR TITLE
Bump controller-runtime to 0.7.0

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
Bump controller-runtime to 0.7.0

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by rashmigottipati — APPROVED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -32,7 +32,7 @@ require (
 	k8s.io/client-go v0.19.4
 	k8s.io/kubectl v0.19.4
 	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/controller-runtime v0.7.0-alpha.7
+	sigs.k8s.io/controller-runtime v0.7.0
 	sigs.k8s.io/controller-tools v0.3.0
```
Comment (estroz) [inline go.mod:36]:
```suggestion
	sigs.k8s.io/controller-tools v0.4.1```

**Comment by estroz** [comment]
@varshaprasad96 there are a bunch more changes needed that #4294 takes care of. You can either copy them over here or close this in favor of that one. Thoughts?

**Comment by varshaprasad96** [comment]
I can close this one, since this is just bumping the versions in go.mod.

**Comment by varshaprasad96** [comment]
Thanks @estroz !

**Pull request closed without merging**